frontend/micro-ui/web/micro-ui-internals/packages/libraries/src/services/atoms/Utils/ParsingUtils.js

```diff
@@ -0,0 +1,53 @@
+import XLSX from 'xlsx';
```

Comment:
add this util in workbench module as well
@nipunarora-eGov

Reply:
@jagankumar-egov Done Jagan